Kiosk/CognitiveServiceApiKeyTester.cs

```diff
@@ -154,6 +157,24 @@ private static async Task<HttpResponseMessage> RequestAndAutoRetryWhenThrottled(
 
             return response;
         }
+
+        public static async Task TestInkRecognizerApiKeyAsync(string key, string apiEndpoint)
+        {
+            bool isUri = !string.IsNullOrEmpty(apiEndpoint) ? Uri.IsWellFormedUriString(apiEndpoint, UriKind.Absolute) : false;
+            if (!isUri)
```

Comment:
Not too sure what the convention is here but I think there should be a space between 163 and 164

Reply:
It looks like the convention laid out in the rest of the file is to not have a space there so I will leave as is for now.